js/index.js

```diff
@@ -0,0 +1,29 @@
+const today = new Date();
+
+let thisYear = today.getFullYear();
+
+console.log(thisYear);
+
+const footer = document.querySelector("#footer");
+
+const copyright = document.createElement("p");
+
+copyright.innerText = "Natalie Petrolino " + thisYear;
+
+footer.appendChild(copyright);
+
+const skills = ["beginner coding", "creative writing", "essay writing", "music production",
+"video editing and production", "dlsr photography and videography" ];
+
+const skillsSection = document.querySelector("#skills");
+
+const skillsList = document.querySelector("#ul"); //check this
```

Comment:
While this works, I think what the assignment was asking you to do is to do this without having to give the `ul` an `id` attribute. One way to do this is to use the `skillsSection` variable and then use the [getElementsByTagName()](https://www.w3schools.com/jsref/met_document_getelementsbytagname.asp) method on it to find the `ul` under it and then select the first one (ie: `[0]`). That way you won't need to specify an `id` for the `ul`.